src/app/core/entity-list/entity-list/entity-list.component.ts

```diff
@@ -301,4 +306,11 @@ export class EntityListComponent<T extends Entity>
     }
     this.addNewClick.emit();
   }
+
+  getFilteredData(filterData: any[]) {
+    this.filteredData = [];
+    filterData.map((item) => {
+      this.filteredData.push(item.record);
+    });
+  }
```

Comment:
I suggest we do this mapping within entity-subrecord-component. I don't see any value of that format having `{ record: Entity }` outside of that component. It will make the interfaces easier to understand and use.

Reply:
@sleidig, Updated.

Reply:
I was talking about something a bit different.

Implemented it myself now. Do have a look at my last commit to understand what I was trying to say :blush: , @brajesh-lab